x-pack/plugins/apm/public/components/shared/Stacktrace/Stackframe.tsx

```diff
@@ -49,6 +49,7 @@ export function Stackframe({
   isLibraryFrame = false
 }: Props) {
   if (!hasLineContext(stackframe)) {
+    console.log('no line context');
```

Comment:
is this console.log intentional? if so maybe the messaging should have more context about lack of line context. 😄

Reply:
Woops! Why did the linter let me commit this???